packages/frontend/src/app/components/login/ConnectWallet.tsx

```diff
@@ -51,6 +51,10 @@ const ConnectWallet = ({ close }: { close: () => void }) => {
     _wallet && setWallet(_wallet);
   }, [walletStatus]);
 
+  useEffect(() => {
+    enrollKeycard.current = false;
+  }, [clientWallet]);
+
```

Comment:
small bug fix: If user changes the connected wallet, we should re-emit the keycard enroll event